### DISCUSSION TITLE
How to use page_separator

### DISCUSSION THREAD

**Discussion by adrianoschommer** [ahrm/sioyek#764] [Q&A] [answered]
I am having difficulties understanding how to use the page_separator. I was expecting to change the colour and width between pages, i.e. upper / lower page (which now is all white -- continuous pages), but I couldn't spot any change. am I misunderstanding the meaning of page_separator?

**Comment by ahrm** [comment]
It is a config option. For example something like this in your prefs_user:
```
page_separator_width 2
```

**Reply by adrianoschommer** [reply]
Yes, but if I change it and save the file, nothing has changed. For example:

## Configure the appearance of page separator
#page_separator_width 5
#page_separator_color	0 0 0

With these parameters, I was expecting a 5-inch separation between pages with a black colour. But I can't see any changes at all on the pdf.

**Reply by ahrm** [reply, marked as answer]
These are commented out. Remove the `#` at the start.
